sdk/containerregistry/azure-containerregistry/samples/async_samples/sample_set_image_properties_async.py

```diff
@@ -0,0 +1,66 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_set_image_properties_async.py
+
+DESCRIPTION:
+    This sample demonstrates setting an image's properties so it can't be overwritten during a lengthy deployment.
+
+USAGE:
+    python sample_set_image_properties_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) CONTAINERREGISTRY_ENDPOINT - The URL of you Container Registry account
+
+    This sample assumes the registry "myacr.azurecr.io" has a repository "hello-world" with image tagged "v1".
+"""
+
+import asyncio
+from dotenv import find_dotenv, load_dotenv
+import os
+
+from azure.containerregistry.aio import ContainerRegistryClient
+from azure.identity.aio import DefaultAzureCredential
+
+
+class SetImagePropertiesAsync(object):
+    def __init__(self):
+        load_dotenv(find_dotenv())
+        self.account_url = os.environ["CONTAINERREGISTRY_ENDPOINT"]
+
+    async def set_image_properties(self):
+        # Create a new ContainerRegistryClient
+        credential = DefaultAzureCredential()
+        audience = "https://management.azure.com"
+        client = ContainerRegistryClient(self.account_url, credential, audience=audience)
+
+        # [START update_manifest_properties]
+        # Set permissions on the image's "latest" tag
+        await client.update_manifest_properties(
+            "library/hello-world",
+            "latest",
```

Comment:
The note above says you're using tag 'v1'

Reply:
Here is setting permissions to the "latest" tag, I'll update the comment in line44 to be more clear.